src/GoodsList.tsx

```diff
@@ -2,14 +2,21 @@ import React from 'react';
 import { Good } from './types/Good';
 
 type Props = {
-  goods: Good[]
+  goods: Good[];
 };
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
     {goods.map(good => (
-      <li key={good.id} data-cy="good">
+      <li
+        key={good.id}
+        data-cy="good"
+      >
         {good.name}
+        -
+        <strong>
+          {good.color}
+        </strong>
```

Comment:
You need to use color for styles, not just write it near the good name. 
![image](https://github.com/mate-academy/react_dynamic-list-of-goods/assets/99403198/9a22eb23-b9f5-4810-9960-948f09ef6d2f)
Here is an example 
![image](https://github.com/mate-academy/react_dynamic-list-of-goods/assets/99403198/048144bb-ba21-46f5-951c-716d8aa05861)